ci/packer/vmbase.json

```diff
@@ -0,0 +1,41 @@
+{
+  "builders": [
+    {
+      "type": "qemu",
+      "iso_urls": [
+        "https://cdimage.debian.org/debian-cd/current/amd64/iso-cd/debian-10.6.0-amd64-netinst.iso"
+      ],
+      "boot_command": [
+        "<esc><wait>",
+        "install <wait>",
+        "auto=true <wait>",
+        "netcfg/get_hostname={{ .Name }} <wait>",
+        "netcfg/get_domain=foo <wait>",
+        "debconf/frontend=noninteractive <wait>",
+        "url=http://{{ .HTTPIP }}:{{ .HTTPPort }}/preseed.cfg <wait>",
+        "<enter><wait>"
+      ],
+      "iso_checksum": "sha256:2af8f43d4a7ab852151a7f630ba596572213e17d3579400b5648eba4cc974ed0",
+      "output_directory": "debian_base",
+      "http_directory": ".",
+      "shutdown_command": "shutdown -P now",
+      "disk_size": "5000M",
+      "cpus": 2,
+      "memory": 1024,
+      "format": "qcow2",
+      "ssh_username": "root",
+      "ssh_password": "root",
+      "ssh_timeout": "40m",
+      "vm_name": "drak-ci",
+      "net_device": "virtio-net",
+      "disk_interface": "virtio",
+      "boot_wait": "10s",
+      "headless": true
+    }
+  ],
+  "provisioners": [
+      { "type": "shell", "script": "install_zfs.sh" },
+      { "type": "shell", "inline": ["wget -q {{user `bundle_url`}} -O drakvuf-bundle.deb", "apt-get install -y ./drakvuf-bundle.deb", "rm drakvuf-bundle.deb"] },
+      { "type": "shell", "inline": ["export DRAK_DOM0_CPU=1", "export DRAK_DOM0_RAM=2048", "update-grub"] }
```

Comment:
Please look at CI logs, this `export`s seemed to be not effective. I.e. it printed different values during `update-grub`

Reply:
hmm, it seems to be OK
```

qemu: Resources dedicated to Dom0 (host system):
--
868 | qemu: -> CPU core(s): 1
869 | qemu: -> Memory: 2048 MB

```